npm/private/test/snapshots/npm_defs.bzl

```diff
@@ -2654,6 +2654,37 @@ def npm_link_all_packages(name = "node_modules", imported_links = []):
         )
         link_targets.append(":{}/js_lib_pkg_a".format(name))
 
+    if is_root:
+        _npm_package_store(
+            name = ".aspect_rules_js/{}/js_lib_pkg_a-alias@0.0.0".format(name),
```

Comment:
Notice how it is adding an additional `npm_package_store` even though we only want a link to a package that already has a store entry.

This is something fixed in https://github.com/aspect-build/rules_js/pull/2177 but I want to add now and only see the diff in https://github.com/aspect-build/rules_js/pull/2177.

Reply:
That functionality might actually be https://pnpm.io/package_json#dependenciesmetainjected, and rules_js just has a different default then pnpm where we always create a store entry for `workspace:`/`link:` deps...